db/migrate/20210916032614_add_default_time_stamp_to_keyword.rb

```diff
@@ -0,0 +1,6 @@
+class AddDefaultTimeStampToKeyword < ActiveRecord::Migration[6.1]
+  def change
+    change_column_default :keywords, :created_at, -> { 'CURRENT_TIMESTAMP' }
+    change_column_default :keywords, :updated_at, -> { 'CURRENT_TIMESTAMP' }
```

Comment:
I think that default timestamp is already current timestamp

Reply:
The problem arises when you do insert_all function with not null validation error. https://github.com/rails/rails/issues/35493